presto-docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -0,0 +1,153 @@
+=================
+Phoenix Connector
+=================
+
+The Phoenix connector allows querying data stored in Phoenix.
+
+Compatibility
+-------------
+
+Connector is compatible with all Phoenix versions starting from 4.13.x.
+
+Installation
+------------
+
+To install a pre-built phoenix, use these directions:
+
+* Download and expand the latest phoenix-[version]-bin.tar.
+* Add the phoenix-[version]-server.jar to the classpath of all HBase region server and master and remove any previous version. An easy way to do this is to copy it into the HBase lib directory.
```

Comment:
Does it refer to Phoenix installation? Or Presto Phoenix connector? I would keep this document only about the connector. Phoenix installation is out of the scope and probably is described somewhere else.